pkg/reconciler/events/cloudevent/cloudeventclient.go

```diff
@@ -34,7 +35,16 @@ type CECKey struct{}
 
 func withCloudEventClient(ctx context.Context, cfg *rest.Config) context.Context {
 	logger := logging.FromContext(ctx)
+
+	// Set a cloud event target to the context if a default sink is defined
+	tektonConfig := config.FromContextOrDefaults(ctx)
+	defaultSink := tektonConfig.Defaults.DefaultCloudEventsSink
+	if defaultSink != "" {
+		ctx = cloudevents.ContextWithTarget(ctx, defaultSink)
```

Comment:
I think this context is passed to the controller, but not all the way down to the reconciler, so the config option may get lost. I might have to move this code to the reconciler code, the same way I had to move add the client to the context.